features/user_sees_static_stuff.feature

```diff
@@ -24,7 +24,7 @@ Feature: User views static pages
   Scenario: I navigate to the Cookies page
     When I go to the home page
     And I follow "Cookies"
-    Then I should be on the cookies page
+    Then I should be on the cookie_policy page
```

Comment:
You should be able to customise this in the paths.rb file:
https://github.com/unboxed/e-petitions/blob/master/features/support/paths.rb